math/vector_cross_product.cpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file
+ *
+ * @brief Calculates the [Cross Product](https://en.wikipedia.org/wiki/Cross_product) and the magnitude of two mathematical vectors from their direction ratios.
+ *
+ *
+ * @details Cross Product of two vectors gives a vector.
+ * Direction Ratios of a vector are the numeric parts of the given vector. They are the tree parts of the
+ * vector which determine the magnitude (value) of the vector.
+ * The method of finding a cross product is the same as finding the determinant of an order 3 matrix consisting
+ * of the first row with unit vectors of magnitude 1, the second row with the direction ratios of the
+ * first vector and the third row with the direction ratios of the second vector.
+ * The magnitude of a vector is it's value expressed as a number.
+ * Let the direction ratios of the first vector, P be: a, b, c
+ * Let the direction ratios of the second vector, Q be: x, y, z
+ * Therefore the calculation for the cross product can be arranged as:
+ * P x Q:
```

Comment:
you can use latex type syntax for better representation. 
See for example, the code [here](https://github.com/TheAlgorithms/C-Plus-Plus/blob/a55e362ee182643a8983244b06179ed022574b9a/numerical_methods/qr_decompose.h#L120)
and its output: [here](https://thealgorithms.github.io/C-Plus-Plus/d2/d3b/namespaceqr__algorithm.html#ab1ac74497ffb2101040cc1efe3546de8)

Reply:
Hello @kvedala ! I don't know latex. The formula for the cross product is pretty simple and I think it's readable without re-writing it in latex code.

Reply:
as described in my comment above, the rendering will not be correct on the documentation website. 
you can at least mark the part as verbatim as below:
```suggestion
 * ```
 * P x Q:
 * 	1	1	1
 *  	a	b	c
 *  	x	y	x
 * ```
```

Reply:
Done!